<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Diagram</class>
 <widget class="QDialog" name="Diagram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelMan">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Средняя частота(%):</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMax">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальная частота(%):</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Минимальная частота(%):</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="outputMan">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="outputMax">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="outputMin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
